src/applications/static-pages/events/components/Results/index.js

```diff
@@ -149,6 +149,18 @@ export const Results = ({
                     </p>
 
                     <div className="vads-u-display--flex vads-u-flex-direction--column">
+                      {event.fieldFacilityLocation?.entity?.entityUrl.path && (
```

Comment:
I'm curious why you didn't include the optional-chaining operator (`?.`) on the last property selection (`path`). I'm not sure if it's possible that `entityUrl` would not be present if `entity` is present, but I do think it's best to treat it as though it is possible.